pom.xml

```diff
@@ -1423,34 +1452,12 @@
             </goals>
             <configuration>
               <includeScope>test</includeScope>
-              <outputFile>${test_classpath_file}</outputFile>
+	      <outputProperty>test_classpath</outputProperty>
             </configuration>
           </execution>
         </executions>
       </plugin>
 
-      <!-- This plugin reads a file into maven property. And it lets us write groovy !! -->
```

Comment:
The result is needed. The groovy plugin not. The trick is 6 lines above in this commit - maven-dependency-plugin can save to file . The outputFile is changed to outputProperty. It does exactly the same thing as the gradle script.

I have double checked it. It does the same thing. The same file is there with the same content.

The advantage of this is therefore smaller maven file and faster build.